spec/graphql/schema/printer_spec.rb

```diff
@@ -390,7 +389,7 @@
 
 enum Choice {
   BAR
-  BAZ @deprecated(reason: "Use \\\"BAR\\\".")
```

Comment:
🤔  Can you help me understand why these escapes are not needed anymore? (Were they wrong to begin with?)

Reply:
Apparently it has to do with the call to inspect here: https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/printer.rb#L164

```
irb(main):031:0> field_or_enum_value.deprecation_reason
=> "use \"this\""
irb(main):032:0> field_or_enum_value.deprecation_reason.to_s
=> "use \"this\""
irb(main):033:0> field_or_enum_value.deprecation_reason.to_s.inspect
=> "\"use \\\"this\\\"\""
```

I don't think its needed 🤔

Reply:
Cool, thanks for the details!